packages/coinjoin/src/client/CoinjoinClient.ts

```diff
@@ -6,21 +6,26 @@ import { CoinjoinPrison } from './CoinjoinPrison';
 import { CoinjoinRound } from './CoinjoinRound';
 import { getNetwork } from '../utils/settingsUtils';
 import { analyzeTransactions } from './analyzeTransactions';
-import type {
+import {
     CoinjoinClientSettings,
     RegisterAccountParams,
     CoinjoinStatusEvent,
     CoinjoinRoundEvent,
     CoinjoinRequestEvent,
     CoinjoinResponseEvent,
 } from '../types';
+import { SessionPhase } from '../enums';
 
-interface Events {
+export interface Events {
```

Comment:
lets move this whole declaration to `types/client.ts` as `CoinjoinClientEvents`
it becomes automatically accessible/exported from index so you ton have to do

`import type { Events as ClientEvents }`

Reply:
done